projects/demo/src/modules/components/table/examples/1/index.ts

```diff
@@ -18,10 +18,12 @@ export default class Example {
         {
             name: 'Alex Inkin',
             balance: 1323525,
+            country: 'UAE',
         },
         {
             name: 'Roman Sedov',
             balance: 423242,
+            country: 'Russia',
```

Comment:
something went wrong 

https://github.com/user-attachments/assets/546c6c8e-9d96-47da-9706-e3c6d309a3a9

Reply:
https://github.com/taiga-family/taiga-ui/issues/7379

Yes, I removed this problem from the description, it remains unsolved

Reply:
Yes, let's do this in another PR. I'm thinking when we first start to drag a column, we should fix current width of all columns. Then we can adjust the column next to the one we drag to the same amount but inversed (when we extend column 2 by 10px we should shrink column 3 by same 10px, when we shrink column 2 — we should extend column 3). That should work, but then we should probably just make a directive for entire table, not for specific columns for this to work.